server/src/main/kotlin/org/javacs/kt/Main.kt

```diff
@@ -39,7 +46,11 @@ fun main(argv: Array<String>) {
         tcpStartServer(it)
     } ?: Pair(System.`in`, System.out)
 
-    val server = KotlinLanguageServer()
+    if (args.fullLog) {
```

Comment:
this will enable something like tracing, when I am debug, I think it is useful, so ..